recruit/organization.go

```diff
@@ -0,0 +1,65 @@
+package zohorecruit
```

Comment:
This package name does not seem to be in alignment with the other files in the package.